src/infra/docs/rustc-ci.md

```diff
@@ -26,14 +26,14 @@ platform’s custom Docker container. This has a lot of advantages for us:
   underlying image (switching from the trusty image to xenial was painless for
   us).
 - We can use ancient build environments to ensure maximum binary compatibility,
-  for example [using CentOS 5][dist-x86_64-linux] on our Linux builders.
+  for example [using CentOS 7][dist-x86_64-linux] on our Linux builders.
```

Comment:
I wonder if there's something we can do to make it harder for this type of info (e.g., ``x86_64-gnu-llvm-13``) to get out of date.

Reply:
Hm, I'm not sure.  How about rewriting it to say "using older Centos releases" instead of saying the specific version?

Reply:
OK, I pushed an updated that removes some of the version-specific stuff.